newIDE/app/src/UI/AlertMessage.js

```diff
@@ -30,8 +30,22 @@ const AlertMessage = ({ kind, children, onHide, renderRightButton }: Props) => (
   <Paper>
     <Line noMargin alignItems="center">
       {kind === 'info' && <Info style={styles.icon} />}
-      {kind === 'warning' && <Warning style={styles.icon} />}
-      {kind === 'error' && <Error style={styles.icon} />}
+      {kind === 'warning' && (
+        <Warning
+          style={{
+            ...styles.icon,
+            color: `orange`,
```

Comment:
Can we use ThemeConsumer and get the color from the theme? In the future themes might want to use other colors.